DayNightQ10.Rmd

```diff
@@ -102,20 +106,35 @@ main_results <- bind_rows(main_results, .id = "Dataset")
 print(main_results)
 ```
 
-## Q10 By Week
-```{r by week, message=FALSE, warning=FALSE}
+## Q10 By Week and Day/Night
+```{r by week, message=FALSE, warning=FALSE, cache = TRUE}
 week_Q10 <- list()
 for(rhds in rh_dataset_names){
   ds_data <- csr_dataset(rhds)
   dsd <- ds_data$data
+  ds_desc <- ds_data$description
   
   column_names <- names(dsd)
   temperature_columns <- column_names[grep("^CSR_T(AIR|[0-9]+)", column_names)]
   rh_ports <- final_results[final_results$Dataset == rhds, ][[2]]
-
-  dsd %>%
+  
+  date <- as.Date(dsd$CSR_TIMESTAMP_END)
+  lon <- ds_desc$CSR_LONGITUDE
+  lat <- ds_desc$CSR_LATITUDE
+  timezone <- attr(dsd$CSR_TIMESTAMP_END, "tzone")
```

Comment:
Wow, very resourceful solution! Probably better would be to use the time zone listed in the `description` table...hmm.